packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -216,12 +218,32 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( { stepIndex } ) =>
 
 const Summary: React.FunctionComponent = () => {
 	const { title, updateTitle, saveTitle } = useTitle();
+	const [ forceSiteNameRender, setForceSiteNameRender ] = useState( false );
+
 	const { sitePrimaryDomain, siteSubdomain, hasPaidDomain } = useSiteDomains();
 	const selectedDomain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
 	const { setDomain, unsetDomain } = useDispatch( LAUNCH_STORE );
-
 	const domainSearch = useDomainSearch();
 
+	// The RegExp check is more relaxed on purpose — chances are that if the title
+	// matches (even it not exactly) the default title, the user would like want
+	// to change it before launch.
+	const isDefaultSiteTitle = new RegExp( DEFAULT_SITE_NAME, 'i' ).test( title );
```

Comment:
We should extract this to some `utils.ts` file and use it also when getting domain search query: https://github.com/Automattic/wp-calypso/blob/master/packages/launch/src/hooks/use-domain-search.ts#L20

Reply:
created `packages/launch/src/utils.ts` containing `isDefaultSiteTitle()` function. Refactored the domain hook and the summary view code